packages/babel-plugin-transform-classes/src/transformClass.js

```diff
@@ -565,11 +566,14 @@ export default function transformClass(
     const closureArgs = [];
 
     if (classState.isDerived) {
-      const arg = classState.extendsNative
-        ? t.callExpression(classState.file.addHelper("wrapNativeSuper"), [
-            t.cloneNode(superName),
-          ])
-        : t.cloneNode(superName);
+      let arg = t.cloneNode(superName);
+      if (classState.extendsNative) {
+        arg = t.callExpression(classState.file.addHelper("wrapNativeSuper"), [
```

Comment:
Can we add pure notation to the helper sources so we don't worry about if the helper needs pure annotation when added?

Reply:
I have thought about it, but the problem is that the annotation should be added to the `CallExpression`, and the `æddHelper` function only returns the callee.